ethcore/engines/validator-set/src/contract.rs

```diff
@@ -165,13 +165,17 @@ impl ValidatorSet for ValidatorContract {
 	fn report_malicious(&self, address: &Address, _set_block: BlockNumber, block: BlockNumber, proof: Bytes) {
 		if let Err(s) = self.do_report_malicious(address, block, proof) {
 			warn!(target: "engine", "Validator {} could not be reported ({}) on block {}", address, s, block);
+		} else {
+			info!(target: "engine", "Reporting malicious validator {} on block {}", address, block);
 		}
 	}
 
 	fn report_benign(&self, address: &Address, _set_block: BlockNumber, block: BlockNumber) {
 		trace!(target: "engine", "validator set recording benign misbehaviour at block #{} by {:#x}", block, address);
 		if let Err(s) = self.do_report_benign(address, block) {
 			warn!(target: "engine", "Validator {} could not be reported ({}) on block {}", address, s, block);
+		} else {
+			info!(target: "engine", "Reporting validator {} on block {} (benign fault)", address, block);
```

Comment:
Seems we already have the warning message inside the `do_report_benign` function.